packages/lab/src/dialog/DialogTitle.tsx

```diff
@@ -13,20 +15,34 @@ import dialogTitleCss from "./DialogTitle.css";
 
 const withBaseName = makePrefixer("saltDialogTitle");
 
-interface DialogTitleProps extends ComponentPropsWithoutRef<"h2"> {
+interface DialogTitleProps {
```

Comment:
These types no longer match now it doesn't extend anything. The change should have been
```suggestion
interface DialogTitleProps extends ComponentPropsWithoutRef<"div"> {
```